Automattic/wp-calypso
# Email: Enable renewals and cancellation for Titan Mail

#### Changes proposed in this Pull Request

This enables renewals and cancellations for Titan Mail.

#### Testing instructions

 - Enable store sandbox
 - Enable the `titan/phase2` flag (append `?flags=titan/phase-2` to the URL)
 - Add Titan Mail to cart and checkout
 - Try to renew, it should work
 - Try to cancel and refund. Again, it should work

New purchase screenshot:

<img width="373" alt="Screenshot 2021-01-11 at 3 34 34 PM" src="https://user-images.githubusercontent.com/13062352/104195201-988e0e80-5422-11eb-8689-c384058eb50b.png">

Renewal screenshot:

<img width="356" alt="Screenshot 2021-01-11 at 3 34 49 PM" src="https://user-images.githubusercontent.com/13062352/104195220-9e83ef80-5422-11eb-9557-3ac7a1c51f5e.png">
